proxy.py

```
@@ -532,22 +554,33 @@ async def _autostart_services_bg() -> None:
+        try:
+            _mongo_client = AsyncIOMotorClient(mongo_url)
+            _mongo_db = _mongo_client["local_llm_server"]
+            from agents.store import AgentStore
+            from tasks.store import TaskStore
+
+            agent_store = AgentStore(db=_mongo_db)
+            task_store = TaskStore(db=_mongo_db)
+            set_agent_store(agent_store)
+            set_task_store(task_store)
+            log.info("MongoDB connected for agent store persistence")
+        except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)